src/chronometric/module/work-value-appraiser.ts

```
@@ -0,0 +1,480 @@
+    return proposals.map((proposal) => {
+      let recommendedMinutes = proposal.normativeMinutes;
+      let basis: 'normative' | 'actual' | 'hybrid' = 'normative';
+
+      switch (policy.mode) {
+        case 'actual':
+          recommendedMinutes = proposal.actualMinutes || proposal.normativeMinutes;
+          basis = 'actual';
+          break;
+
+        case 'blended':
+          const ratio = policy.blendRatio ?? 0.5;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)